src/api.ts

```diff
@@ -100,33 +100,76 @@ export async function deleteImage(hash: CommitHash) {
 	}
 }
 
+const startingContainers: Map<CommitHash, Promise<ContainerInfo> > = new Map();
```

Comment:
would you be willing to add additional state into `apiState` so we can see it all at a glance? do you think it's better to have it spread around the file like this?

Reply:
i keep forgetting about apiState. I'll move it there.

Reply:
done